modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/InviteUsersMVCResourceCommand.java

```diff
@@ -299,7 +300,8 @@ private Role _getRole(int roleValue, ThemeDisplay themeDisplay)
 	}
 
 	private void _sendEmail(
-			long ctCollectionId, long receiverUserId, ThemeDisplay themeDisplay)
+			CTCollection ctCollection, long ctCollectionId, long receiverUserId,
+			ThemeDisplay themeDisplay)
```

Comment:
We don't need both `ctCollection` and `ctCollectionId` in the params. We can get the object from the id or you can pass in the object if you don't want to do another fetch.